### PR TITLE
refactor: moved scraping utilities to services/scraper folder

### LINKED ISSUE #74: Refactor Code
- [ ] Check and add docstrings, typehints, comments
- [ ] Move `utils/scraping.py` (could rename it to scraper.py) and `related functions` to a new folder `services/scraper` and access it from there.

[Add things that can be refactored such as - function calls(flow), function/variable names, introduce constants/env variables, use of data-structures to represent/handle information in application]

### PR DESCRIPTION
**Pull request by karelumair**
Added typehints and docstrings

- [X] Check and add docstrings, typehints, comments
- [X] Move utils/scraping.py (could rename it to scraper.py) and related functions to a new folder services/scraper and access it from there.

closes #74

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -44,8 +44,11 @@ def create_app(env: str = "test", context: str = "main") -> Flask:
 
     # Health check
     @app.route("/health/")
-    def health_check():
-        return {"status": "ok"}
+    def health_check() -> dict:
+        try:
+            return {"status": "ok"}
+        except Exception as exp_err:
+            return {"Error": str(exp_err)}
```
Comment (pratikGhodke1) [inline app.py]: No need to add this in try except. Keep it as it is.

Comment (karelumair) [reply app.py]: ok

**Comment by pratikGhodke1** [comment]
This looks good. Can you check of other possibilities where code can be modified to make it less coupled. OR code logic can be reduces OR kind of better ways to do things. Also check out high priority issues first.

**Comment by karelumair** [comment]
Done with all changes.

File: pyproject.toml [hunk 2]
```diff
@@ -25,7 +25,7 @@ pytest-celery = "^0.0.0"
 flask-apscheduler = "^1.12.4"
 
 [tool.pylint.messages_control]
-disable = "E1101, E0611, W0703, C0301, W0223, W0613, R0903, W0621"
+disable = "E1101, E0611, E0401, W0703, C0301, W0223, W0613, R0903, W0621"
```
Comment (pratikGhodke1) [inline pyproject.toml]: Why we added this error in ignore errors list? Can add what is this error and where did it occur. Let's take a look at all these error codes to see if we can correct those!

File: services/scraper/scraper.py [hunk 3]
```diff
@@ -2,41 +2,65 @@
...
+    """This function scrape quotes and authors data at a time.
+
+    Args:
+        single_page (bool): bool to scrape only single page
+
+    Returns:
+        dict: dict of no. of fetched records
+    """
+    quote_status = scrape_quotes(self, single_page)
     author_status = scrape_authors(self)
```
Comment (pratikGhodke1) [inline services/scraper/scraper.py]: You might need to handle error here! or have to handled them inside these functions?

**Comment by karelumair** [comment]
Done.

**Review by pratikGhodke1 — APPROVED** [review]

**Pull request merged**